<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup_dialog</class>
 <widget class="QDialog" name="signup_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#signup_dialog{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 5, 68, 255), stop:1 rgba(10, 55, 255, 255));}</string>
  </property>
  <widget class="QLabel" name="registration_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>-30</y>
     <width>531</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 28pt &quot;Arial&quot;;color:rgb(20, 255, 255)</string>
   </property>
   <property name="text">
    <string>REGISTRATION FORM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
font: 12pt &quot;Arial&quot;;
color: rgb(39, 255, 237);
border: none;
border-bottom: 4px solid rgba(46,82,101,200)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="surename_input">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
color: rgb(39, 255, 237);
border: none;
border-bottom: 4px solid rgba(46,82,101,200);
font: 12pt &quot;Arial&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_input">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
color: rgb(39, 255, 237);
border: none;
border-bottom: 4px solid rgba(46,82,101,200);
font: 12pt &quot;Arial&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwd_input">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
color: rgb(39, 255, 237);
border: none;
border-bottom: 4px solid rgba(46,82,101,200);
font: 12pt &quot;Arial&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rpasswd_input">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>550</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
color: rgb(39, 255, 237);
border: none;
border-bottom: 4px solid rgba(46,82,101,200);
font: 12pt &quot;Arial&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 12pt &quot;Arial&quot;;color:rgb(34, 255, 255)</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="surename_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 12pt &quot;Arial&quot;;color:rgb(34, 255, 255)</string>
   </property>
   <property name="text">
    <string>Surename</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 12pt &quot;Arial&quot;;color:rgb(34, 255, 255)</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwd_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 12pt &quot;Arial&quot;;color:rgb(34, 255, 255)</string>
   </property>
   <property name="text">
    <string>Password*</string>
   </property>
  </widget>
  <widget class="QLabel" name="requirements_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>810</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 8pt &quot;Arial&quot;;color:rgb(34, 255, 255)</string>
   </property>
   <property name="text">
    <string>*at least 8 characters: number; uppercase and lowercase letter required</string>
   </property>
  </widget>
  <widget class="QLabel" name="rpasswd_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 12pt &quot;Arial&quot;;color:rgb(34, 255, 255)</string>
   </property>
   <property name="text">
    <string>Password Retyped</string>
   </property>
  </widget>
  <widget class="QLabel" name="failure_info">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>670</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 12pt &quot;Arial&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signup_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>620</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
font: 87 16pt &quot;Arial&quot;;
color: rgb(39, 255, 237);</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
font: 87 12pt &quot;Arial&quot;;
color: rgb(39, 255, 237);</string>
   </property>
   <property name="text">
    <string>Already have an account? Click here to log in or to visit main page.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
